ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/boolean/activity-boolean-checkbox-answer/activity-boolean-checkbox-answer.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ActivityBooleanQuestionBlock } from '../../../../../models/activity/activityBooleanQuestionBlock';
+
+@Component({
+    selector: 'ddp-activity-boolean-checkbox-answer',
+    template: `
+        {{ block.answer }}
+        <mat-checkbox
+            [checked]="block.answer"
+            [disabled]="readonly"
+            [disableRipple]="true"
+            (change)="onValueChange()"
+        >
+            <p
+                [innerHTML]="
+                    block.answer ? block.trueContent : block.falseContent
```

Comment:
I suppose, the component CHECKBOX view should not use **trueContent**/**falseContent** properties at all.

Reply:
Sounds good